packages/ui/src/app/components/SideBar.tsx

```diff
@@ -75,11 +77,13 @@ export const SideBarContent = () => {
           {/*    Overview*/}
           {/*  </NavigationLink>*/}
           {/*</NavigationLinksItem>*/}
-          <NavigationLinksItem>
-            <NavigationLink to={ProfileRoutes.profile} icon={<MyProfileIcon />}>
-              My profile
-            </NavigationLink>
-          </NavigationLinksItem>
+          {size !== 'xxs' && size !== 'xs' && (
```

Comment:
Here I think the profile link not showing shouldn't be based directly on `size`. What about adding `supportTransactions: boolean` to the `UseResponsive` context ? For now in the provider the condition could be something like:
```ts
supportTransactions = screen.width >= 768
```
This way we could change the condition later on if necessary.

So this would become:
```suggestion
          {supportTransactions && (
```